<launch>

    <node pkg="computer_vision" type="project_setup_node.py" name="ProjectSetup" />
    
    <node pkg="computer_vision" type="activation_node.py" name="ActivationTest" />

    <node pkg="computer_vision" type="ur_node.py" name="UR" />

  

</launch>
